### DISCUSSION TITLE
Recreating results from 'On the effective reconstruction of expectation values from ab initio quantum embedding'

### DISCUSSION THREAD

**Discussion by tommontgomery450** [BoothGroup/Vayesta#170] [General]
Hi I'm looking to use Vayesta to recreate the results for The Chlorine Dimer in the paper 'On the effective reconstruction of expectation values from ab initio quantum embedding' found [here](https://kclpure.kcl.ac.uk/ws/portalfiles/portal/201474006/Expectation_Values_from_Quantum_Embedding_6_.pdf). I managed to get the energy for the unoptimised chemical potential in all cases except for the energy from the RDMs of the T-amplitude projected global wave function (E[(γ, Γ)[Ψx]] from Sec. III D).. I'm not sure how to be setting up the EWF or what method to be calling to get this energy or if you have any code to share on how to calculate this for the Chlorine Dimer example.

Tom

**Comment by basilib** [comment]
The method used is [here](https://github.com/BoothGroup/Vayesta/blob/ab6c5080648dc0e9bc636fa3af6e9321e15d92aa/vayesta/ewf/ewf.py#L527C9-L527C68). To get E[(γ, Γ)[Ψx]] both arguments should be set to True.

This will only work if CCSD is used as a fragment solver. However for the Chlorine dimer, there is no truncation of excitation rank, and the result will match FCI.
